instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientCommonAttributesGetter.java

```diff
@@ -8,40 +8,18 @@
 import javax.annotation.Nullable;
 
 /** An interface for getting attributes common to database clients. */
+@Deprecated
 public interface DbClientCommonAttributesGetter<REQUEST> {
```

Comment:
I returned this class to how it existed in the last release (`v2.9.0`), so the method removals in this class are ok if we merge before `v2.10.0`